contracts/OlasMech.sol

```diff
@@ -20,8 +20,8 @@ interface IMechMarketplace {
 /// @dev A Mech that is operated by the multisig of an Olas service
 abstract contract OlasMech is Mech, IErrorsMech, ImmutableStorage {
     event MaxDeliveryRateUpdated(uint256 maxDeliveryRate);
-    event Deliver(address indexed sender, uint256 requestId, bytes data);
-    event Request(uint256 requestId, bytes data);
+    event Deliver(address indexed mech, address indexed mechServiceMultisig, uint256 requestId, bytes data);
+    event Request(address indexed mech, uint256 requestId, bytes data);
```

Comment:
Adding mech addresses into events to make all request / deliver events symmetrical along with MM requests coming with signatures.